domains/service/Cargo.toml

```diff
@@ -63,8 +65,10 @@ sp-subspace-mmr = { version = "0.1.0", path = "../../crates/sp-subspace-mmr" }
 sp-transaction-pool = { git = "https://github.com/subspace/polkadot-sdk", rev = "5871818e1d736f1843eb9078f886290695165c42" }
 subspace-core-primitives = { version = "0.1.0", path = "../../crates/subspace-core-primitives" }
 subspace-runtime-primitives = { version = "0.1.0", path = "../../crates/subspace-runtime-primitives" }
+subspace-service = { version = "0.1.0", path = "../../crates/subspace-service" }
```

Comment:
This is very problematic. You add a gigantic crate as a dependency to another gigantic crate. And the only purpose is to access `ConsensusChainSyncParams`, which most likely doesn't belong to `subspace_service` in the first place.

Reply:
I can get rid of this dependency by moving the struct. However, I would need to add `sc-consensus-subspace` dependency to `domain-operator` (block import streams dependency in the struct). Is it better?

Reply:
Not really, domains should not rely on Subspace consensus code either